README.md

```diff
@@ -10,12 +10,25 @@ See the [documentation](https://algolia.com/doc/tools/cli/) for setup and usage
 
 ## Installation
 
+### From repository
```

Comment:
I would rename section to "Build from Source"

and maybe include the "git clone" for the repo first, so that it is explicitly clear that you gotta have the files on hand to make 'em

Reply:
Way better! 🙏